server/src/database/connection.js

```diff
@@ -1,6 +1,7 @@
 import mongoose from 'mongoose';
 import { DB_URL } from '../app/appConstants';
+import mongoConfig from './mongo.config';
```

Comment:
All file name are camelCased, then why exception for this?

Reply:
This was just to make it clear that that is a config file, like `jest.config.js`. I will change this to camelCase.